### ISSUE TITLE
Incorrect sources showing up in inferred treats EPC modal

### ISSUE THREAD

**Issue by mbrush** [NCATSTranslator/Feedback#724]
Search on test for what may treat breast cancer: https://ui.test.transltr.io/main/results?l=Breast%20Cancer&i=MONDO:0007254&t=0&r=0&q=a6d28087-f193-441c-ade3-26b04fdd0dd1 

Get Mitoxantrone as a result that says "6 paths that treat Triple-negative Breast Carcinoma" (side note: I have always disliked this phrase "X paths that treat Disease X" - it is non-sensical). 
 
When I click on the Inferred 'treats' edge in the Paths section, the modal shows **several different Knowledge Sources**:
 
![image](https://github.com/NCATSTranslator/Feedback/assets/5184212/3a5870fb-a450-4c5e-989e-d40cd6e7e5bf)
 
**There should only be one source here  - BTE** - as they are the one who predicted the inferred result). If these other sources provided a 'treats' edge, there should be additional top level / direct treats edges showing up in the display.  
 
Are these other sources ones that provided edges in support paths?  These should not be showing up in the modal for the predicted treats edge.

**Comment by gprice1129** [comment]
@mbrush we believe this is a bug caused by BTE being marked as a curated source in the infores catalog v3.6.0.

**Comment by dnsmith124** [comment]
@gprice1129 regarding the number of sources on this result, looking at the raw response from the BE all of these sources are attached to the edge itself.

**Comment by mbrush** [comment]
Not sure I follow . . . if the other sources listed for this edge (Chembl, Hetionet, AEOLUS, etc) also provide the edge stating that 'Mitoxantrone treats Breast Cancer', why are these edges not showing up in the 'Lookup' section as top level results?  

Is it because the behavior implemented by the UI is to merge all 'curated' edges into one graphic edge in the 'Lookup' section, but because the BTE edge is erroneously tagged as 'curated', and simultaneously interpreted to be 'Inferred' b/c of its support graphs, the UI doesn't know what to do  - so it creates an "Inferred" edge graphic, but then lumps all the other curated sources in with BTE in the modal on this Inferred edge? 

More importantly, how do we fix this.  Is it just a matter if correcting the 'crated' flag on BTE in the infores catalog?  Will this fix things in the UI?

**Comment by dnsmith124** [comment]
That's correct Matt. The solution is for BTE to not erroneously tag inferred edges as curated. If that were implemented here, you'd see a second curated direct path of Mitoxantrone treats Breast Cancer above the inferred one from BTE, and the BTE edge would not have all those extra knowledge sources.

**Comment by mbrush** [comment]
Got it.  As of  v4.1.1 the knowledge type tag on infores:biothings-explorer says `other', not `curated`.  ounds like the issue should be fixed once you update to use this latest version. Let's keep this issue open until then, so we can confirm this does it.
